src/@types/Orbis.d.ts

```diff
@@ -0,0 +1,623 @@
+declare module '@orbisclub/orbis-sdk'
+
+declare interface IOrbisConstructor {
```

Comment:
as a general thing we tend to split type declaration files that apply to one component only into smaller files so would be easier to track and follow. as for eg we could have splitted this into _types.tsx files for the Orbis context types  and grouping them inside folders, and one new file under _types.tsx under the DirectMessages component. 
See as reference the `@context/Prices/` or `@context/MarketMetadata`